index.html

```diff
@@ -5026,19 +5057,23 @@ <h2>Validation</h2>
       </p>
 
       <section class="informative">
-        <h3>Credential Subject</h3>
+        <h3>Holder</h3>
```

Comment:
<strike>These are not interchangeable in this way.  *Maybe* an added `Holder` section will work.  Replacing the `Credential Subject` section does not work for me at all.</strike>

...

Now I've read further along, and I see that you did add a `Holder` section.  You just confused me by putting this new section ahead of the existing `Credential Subject` section, which order doesn't seem logical to me.  What is this ordering based on?